documents/design/new_member_project.md

```diff
@@ -0,0 +1,71 @@
+
+What is the problem you are trying to solve? Why is it important to solve this problem?
+What is the end goal of this project? This section should similar to the body of the GitHub
+issue but include more details about it. This will be a qualitative description of the
+project. Use as many details you can to describe the requirements of the project.
+*(This is similar to the Motivation section of the other design documents.)*
+
+## Proposed Solution
+
+- How are you going to solve this problem?
+- What are the steps you need to take to complete the project?
+- Break down the problem above into smaller, individual components with specific metrics of success
+- You can think of this section as a quantitative description of the project
+- Each bullet should represent a single action
+    - Use sub-bullets to provide more details and justifications for each step
+- *(Replace these bullet points with your own)*
+
+_It's OK if your approach changes later in the project as you learn more. Treat this like
+a road map that you can come back to to figure out what to work on next. If you aren't sure about the
+technical details, make sure your ideas are clear. It's also OK to come up with a couple solutions
+and decide which to pursue later. There should be some justification with your solution (i.e. how 
+does each step address part of the problem above)._
+
+## Questions & Research
+
+Are there things you are unsure about or don't know? What do you need to research to be able to
+complete this project? If you need information from the mechanical or electrical subteams,
+be sure to describe that here. If your solution requires more research to implement, descibe
+what kind of topics you need look at. Link any research papers/articles that you find here.
+
+**Of course, if you have _any_ questions or concerns, feel free to bring them up at any time.
+If you ever feel lost or don't know what to do or work on, Oswin or any old software member
+can help you out. You are not expected to know every technical detail about the project. Expressing
+what you don't know will make it easier to do research and ask for help.** 
+
+## Overall Scope
+
+### Affected Packages
+
+- What parts of the software will you have to change (if any)?
+- Which packages are relevant to the success of the project?
+- You're first step of the project should be reviewing the relevant packages and code
+
+### Responsibilities
+
+- If you have more than one person working on this project, what will each person do?
+- My Name: one of actions described in the solution
+- *(Remove this section if you are doing this alone.)*
+
+### Schedule
+
+Week 1 (Date): What do expect to get done each week?
+
+Week 2 (Date): Think about how long each portion of the solution will take
+
+Week 3 (Date): This will be used as a rough guide for keeping track of your progress, 
+but it's ok if you fall behind schedule. 
+
+Week 4 (Date): Include an estimate for when your code/result will be ready to review
```

Comment:
Instead of a week-by-week estimate, I think breaking up the project into like sub tasks and then giving per-week estimates would probably be a better method of encouraging them to think about how to plan the project.